pkg/cloud/gcp/gcp.go

```diff
@@ -407,6 +423,36 @@ func (c *GcpCloud) StopGCPNode(nodeConfig models.NodeConfig, clusterName string)
 	return nil
 }
 
+// AddFirewall adds firewall into an existing project in GCP
+func (c *GcpCloud) AddFirewall(monitoringHostPublicIP, networkName, projectName string) error {
+	firewallName := fmt.Sprintf("%s-%s-monitoring", networkName, strings.ReplaceAll(monitoringHostPublicIP, ".", ""))
```

Comment:
nit: AddFirewall is generic function to add FW rules, `monitoring` should not be hardcoded in FWname